CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ recommendations of [keepachangelog.com](http://keepachangelog.com/).
 
 - None
 
+### Dependencies
+- Drop support for Rails 6.1, which [reached EoL on 2024-10-01][2]
```

Comment:
Empty line is needed and not sure if these `[2]` are needed

Reply:
i copied this pattern from an [older `CHANGELOG.md` message](https://github.com/paper-trail-gem/paper_trail/blob/ec22c85212143b44a14f78f51f3bcb6f9d4ec41e/CHANGELOG.md#L103-L105
)

```markdown
- Drop support for Rails 6.0, which [reached EoL on 2023-06-01][2]
- Drop support for Ruby 2.7, which [reached EoL on 2023-03-31][3]

```

Reply:
> Empty line is needed 

fixed.